test-cases/tests/upgrade/n01-verify-customer-config-reconcile-upgrade-logic.md

```diff
@@ -0,0 +1,41 @@
+---
+estimate: 2h
+require:
+  - N01
+---
+
+# N01 - verify customer config reconcile upgrade logic
+
+## Prerequisites
+
+1. Login via `oc` as a user with **cluster-admin** role (kubeadmin):
```

Comment:
A user from dedicate-admins group should also have the permissions we need here, but getting an oc session would have more steps, we would need to obtain a token through UI...